manifests/o/OSGeo/QGIS/3.22.3/OSGeo.QGIS.locale.en-US.yaml

```
@@ -0,0 +1,27 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=NVS1.5-1-19041-1320
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: OSGeo.QGIS
+PackageVersion: 3.22.3
+PackageLocale: en-US
+Publisher: QGIS Development Team
```

Welp there is also this
```suggestion
Publisher: QGIS.org
```